src/lib/nexus-schema-stateful/stateful-nexus-schema.ts

```diff
@@ -28,6 +29,49 @@ export interface NexusSchemaStatefulBuilders {
   idArg: typeof NexusSchema.idArg
   extendType: typeof NexusSchema.extendType
   extendInputType: typeof NexusSchema.extendInputType
+  /**
+   * Add a GraphQL.js type in your Nexus schema
+   */
+  importType: {
+    /**
+     * Add a GraphQL.js scalar type and (optionally) expose it as a method in your definition builders.
+     * Check the second overload if you're not adding a scalar type.
+     * 
+     * @example
+     * 
+     * ```ts
+     * import { schema } from 'nexus'
+     * import { GraphQLDate } from 'graphql-iso-date'
+     *
+     * schema.importType(GraphQLDate, 'date')
+     *
+     * schema.objectType({
+     *  name: 'SomeObject',
+     *  definition(t) {
+     *   t.date('createdAt') // t.date() is now available (with types!) thanks to `importType`
+     *  },
```

Comment:
Indentation 🙈 

Any prettier integration for jsdoc code blocks? Quick google search didn't turn up anything obvious.

Reply:
Thanks! None that I know about